docs/src/site/twiki/EntitySLAAlerting.twiki

```diff
@@ -0,0 +1,57 @@
+---++Entity SLA Alerting
+
+Falcon supports sla in feed and process.
+
+Types of sla supported for feed:
+
+   1.slaLow
```

Comment:
Hi @sepreece 

Thanks for the review I also thought we should add what slaLow and slaHigh but it should go EntitySpecification page not here.I have already provided the link in the documentation.

Same is the case with process.